com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/OperationsApi.java

```diff
@@ -91,9 +91,10 @@
         }) }, tags = {})
     @ApiResponses(value = { @ApiResponse(code = 200, message = "OK", response = Operation.class, responseContainer = "List") })
     public ResponseEntity<List<Operation>> getOperations(@PathVariable(PathVariables.SPACE_GUID) String spaceGuid,
+                                                         @RequestParam(name = PathVariables.MTA_ID, required = false) String mtaId,
                                                          @RequestParam(name = "last", required = false) Integer last,
```

Comment:
Why this "last" and "state" are not presented as contants but the MTA_ID is ?

Reply:
will add them